virtual_rainforest/models/animals/dummy_plants_and_soil.py

```diff
@@ -0,0 +1,52 @@
+"""The ''dummy plands and soil'' classes provides toy plant and soil module functionality 
+that  are required for setting up and testing the early stages of the animal module.
+
+"""  # noqa: #D205, D415
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from virtual_rainforest.core.logger import LOGGER
+from virtual_rainforest.models.animals.constants import ENERGY_DENSITY
+
+
+class PlantCommunity:
+    """This is a dummy class of plant cohorts for testing the animal module."""
+
+    def __init__(self, mass: float, position: int) -> None:
+        """The constructor for Plant class."""
+        self.mass = mass
+        """The mass of the plant cohort [kg]."""
```

Comment:
I'm trying to stick with SI units where I can, so kg for mass. The issue is that the allometric equations I have are in a wild tangle of forms. When we get to dialling in the model parameters, I'll get to standardizing them. Right now, the actual scaling functions account for the discrepancies.

Reply:
I think this was already discussed, but sticking to `pint` objects whenever you're dealing with scalars that have units will solve this issue. Not something to change without further discussion, but just an option to consider to avoid possible errors due to the inconsistency of units. I don't know if that would affect the performance significantly, though.